tools/workflow/clean_aws_resources.go

```diff
@@ -97,11 +105,11 @@ func terminateEc2Instances() {
 	}
 }
 
-func destroyLoadBalancerResource() {
+func destroyLoadBalancerResource(awsRegion string) {
 	// Set up aws go sdk session
 	// Only using default environment variables instead of loading other metadata from session.NewSessionWithOptions
 	//Documents: https://docs.aws.amazon.com/ja_jp/sdk-for-go/v1/developer-guide/configuring-sdk.html
-	testSession, err := session.NewSession()
+	testSession, err := session.NewSession(aws.NewConfig().WithRegion(awsRegion))
```

Comment:
Can we reuse the session and just pass it into these functions instead of the region string?

Reply:
Yes. We can delete the getting the environment variables and use the configure credentials with Session. Fixed it with newest commit